packages/grpc-protobufjs/src/index.ts

```diff
@@ -97,14 +97,14 @@ function createMethodDefinition(method: Protobuf.Method, serviceName: string, op
     responseSerialize: createSerializer(method.resolvedResponseType as Protobuf.Type),
     responseDeserialize: createDeserializer(method.resolvedResponseType as Protobuf.Type, options),
     // TODO(murgatroid99): Find a better way to handle this
-    originalName: _.camelCase(method.name)
+    originalName: method.name
   };
 }
 
 function createServiceDefinition(service: Protobuf.Service, name: string, options: Options): ServiceDefinition {
   const def: ServiceDefinition = {};
   for (const method of service.methodsArray) {
-    def[method.name] = createMethodDefinition(method, name, options);
+    def[_.camelCase(method.name)] = createMethodDefinition(method, name, options);
```

Comment:
The changes in this file shouldn't make a difference. The whole point of the `originalName` property is that both variants are usable as method names in clients and servers. So I would prefer not to change this.